### PR TITLE
French translation of documentation

### PR DESCRIPTION
**Pull request by francisek**
#54 A less word to word translation than @Varpie did.

### CODE REVIEW

File: docs/fr/README.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+
+#### Embarquer Un QrCode
+
+Vous pouvez embarquer un qrcode dans un courriel pour permettre à vos utilisateurs de le flasher. Voici un exemple pour mettre ceci en œuvre dans Laravel :
+
+	// Dans un template blade.
+	<img src="{!!$message->embedData(QrCode::format('png')->generate('Embarquez-moi dans un courriel!'), 'QrCode.png', 'image/png')!!}">
+
+<a id="docs-usage"></a>
+## Usage
```
Comment (Varpie) [inline docs/fr/README.md]: « Utilisation » is less word to word, and actually more correct.

File: docs/fr/README.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+Voici la liste des options supportées pour la méthode `errorCorrection`.
+
+| Correction d'Erreur | Capacité De Correction |
+| --- | --- |
+| L | 7% de redondance. |
+| M | 15% de redondance. |
+| Q | 25% de redondance. |
+| H | 30% de redondance. |
+
+>L'élévation du niveau de correction d'erreur se fait au détriment de la taille du QrCode et de la quantité de données qu'il peut stocker. Pour en savoir plus, consultez [error correction](http://en.wikipedia.org/wiki/QR_code#Error_correction) (en anglais).
```
Comment (Varpie) [inline docs/fr/README.md]: You could have easily linked to the french page here (https://fr.wikipedia.org/wiki/Code_QR#Correction_d.27erreur)

Comment (francisek) [inline docs/fr/README.md]: Yes but, the french page only provides values that are in the documentation.

File: docs/fr/README.md (hunk 1 above)
Comment (francisek) [inline docs/fr/README.md]: « Utilisation » and « Usage » could both be used. I prefer « Usage » in this case because it has a technical connotation.

**Comment by SimplyCorey** [comment]
@Varpie @francisek Thanks to both of you for making a translation.  I see two different versions of the French translation.  Which one should I merge in?

**Comment by Varpie** [comment]
@SimplyCorey I think @francisek's one is better, as he said mine is more word to word (also, I forgot to translate a few things, the list at the top for instance).

**Comment by SimplyCorey** [comment]
@francisek @Varpie Thanks again guys.

**Pull request merged**